src/sbt-test/ref/example-library/build.sbt

```diff
@@ -1,10 +1,13 @@
 ThisBuild / scalaVersion := "2.13.6"
 ThisBuild / organization := "com.example"
 
-lazy val hello = (project in file("."))
+lazy val hello = project
+  .in(file("."))
   .settings(
     name := "Hello",
-    libraryDependencies += "com.typesafe.play" %% "play-json" % "2.9.2",
-    libraryDependencies += "com.eed3si9n" %% "gigahorse-okhttp" % "0.5.0",
-    libraryDependencies += "org.scalatest" %% "scalatest" % "3.2.7" % Test,
+    libraryDependencies ++= Seq(
+      "com.softwaremill.sttp.client4" %% "core" % "4.0.0-M2",
+      "com.lihaoyi" %% "ujson" % "3.1.2",
+      "org.scalatest" %% "scalatest" % "3.2.16" % Test
+    )
```

Comment:
or, if you want to avoid that for some reason, at least MUnit instead of ScalaTest

Reply:
I don't have a big reason why I didn't just use the toolkit. Mainly because I thought it was nice to show exactly what we were adding since the guide sort of slowly introduces a testing framework, then the other deps. Then they break the other deps out. So if we're just including the toolkit, then you might lose out on some of that. Although, maybe not. Let me go back through and double-check and if I can still illustrate the same things with it I'll switch. Especially swapping out scalatest for munit.